haskell/test/Test/Ligo/RegistryDAO.hs

```diff
@@ -478,7 +482,7 @@ tokenTransferType contractAddr fromAddr toAddr = DAO.Token_transfer_type DAO.Tok
       , tiTxs =
         [ FA2.TransferDestination
           { tdTo = toAddr
-          , tdTokenId = unfrozenTokenId
+          , tdTokenId = frozenTokenId
```

Comment:
Am I correct that here and in other places you replaced this to be able to still make transfers on the DAO itself?

AFAIU this limits them to be executed by the `admin` as `from`, which is not very useful and I'd say an uncommon usage (although possible).
Since most transfers are going to happen on the (now-external) governance token I'd say that's what we should test for instead.

Reply:
I have changed this to a hard coded token id of `0`.